packages/unmock-cli/src/commands/curl.ts

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+import { unmock } from "unmock-node";
+
+export const makeHeaders = (headers: string[]) =>
+  headers
+    .map(h => ({
+      [h.substring(0, h.indexOf(":")).trim()]: h
+        .substring(h.indexOf(":") + 1)
+        .trim(),
+    }))
+    .reduce((a, b) => ({ ...a, ...b }), {});
+
+export const curlInternal = async (url: string, options: any) => {
+  await unmock({
+    ...(options.signature ? { signature: options.signature } : {}),
+  });
+  const { data } = await axios({
+    ...(options.data ? { data: JSON.parse(options.data) } : {}),
+    ...(options.headers ? { headers: makeHeaders(options.headers) } : {}),
+    method: options.request || "GET",
+    url,
+  });
+  return data;
+};
+
+export const curl = async (url: string, options: any) => {
```

Comment:
We probably don't want to do network calls from the CLI `curl` anymore, maybe the internal `curl` could use `unmock-fetch` to mock network calls with in-memory faking. But this is fine now as is!

Reply:
OK